api/_utils.ts

```diff
@@ -38,6 +39,10 @@ export const DEFAULT_GAS_MARKUP = 0;
 
 export const HUP_POOL_CHAIN_ID = Number(REACT_APP_HUBPOOL_CHAINID || 1);
 
+export const FLAT_RELAY_CAPITAL_FEE = REACT_APP_FLAT_RELAY_CAPITAL_FEE
+  ? Number(process.env.REACT_APP_FLAT_RELAY_CAPITAL_FEE)
+  : 0;
```

Comment:
```suggestion
export const FLAT_RELAY_CAPITAL_FEE = Number(REACT_APP_FLAT_RELAY_CAPITAL_FEE ?? 0);
```

Is there a valid case for ever having a 0% relayer capital fee? How about we stick to something like 3bps and then override via REACT_APP_FLAT_RELAY_CAPITAL_FEE?

Reply:
i like that